app/graphql/types/supporter_type.rb

```diff
@@ -0,0 +1,8 @@
+class Types::SupporterType < Types::BaseObject
+  field :id, ID, null: false
+  field :name, String, null: false
+  field :email, String, null: false
```

Comment:
Maybe we can eventually add a phone number as well, in case a volunteer would like to be contacted by the admin

Reply:
Sure! That would be pretty easy to add in down the road (or now if we want to!)